ci/tox/Dockerfile

```diff
@@ -33,25 +33,9 @@ ENV PYTHON_VERSION=${python_version}
 ENV PYTHON=python${python_version}
 RUN apk add --no-cache gcc musl-dev # both packages are required to install mypy
 
-ARG user_id
```

Comment:
Crap, on my machine I still have the files visible as belonging to root after local-run.sh :///

How about you do these steps optional indeed? So that for local runs, the user of the same UID/GID as the host user is still created (so that `userns-remap` magic is not needed)